PasswordVerifierTest/PasswordVerifier_VerifyPasswordShould.cs

```diff
@@ -0,0 +1,66 @@
+using PasswordVerifier;
+using Xunit;
+
+namespace PasswordVerifierTest
+{
+    public class PasswordVerifier_VerifyPasswordShould
+    {
+        [Fact]
+        public void VerifyPassword_InputIsqwe123_ThrowsExceptionNoMinimum9Characters()
+        {
+            //act
+            static void act() => Verify.VerifyPassword("qwe12345");
```

Comment:
use variable for invalid password & name it something like `invalidLengthPassword` or `shortPassword` or `invalidPassword`